CONTRIBUTING.md

```diff
@@ -14,17 +14,14 @@ file.
 
 * Download and install the latest stable version of IntelliJ
   - [IntelliJ Downloads](https://www.jetbrains.com/idea/download/)
-  - either the community edition (free) or Ultimate will work.
+  - Either the community edition (free) or Ultimate will work.
 * Start IntelliJ
 * In the Project Structure dialog (`File | Project Structure`), select "Platform Settings > SDKs" click the "+" sign at the top "Add New SDK (Alt+Insert)" to configure an IntelliJ Platform Plugin SDK
-  - point it to the directory of your downloaded IntelliJ Community Edition installation (e.g, `IntelliJ IDEA CE.app/Contents` or `~/idea-IC-183.4886.37`)
-  - change the name to `IntelliJ IDEA Community Edition`
-  - extend it with additional plugin libraries (only until Android Q is released)
-    - plugins/android/lib/android.jar
-    - plugins/gradle/lib/gradle-common.jar
-* One-time Dart plugin install - first-time a new IDE is installed and run you will need to install the Dart plugin.  `Configure | Plugins` and install the Dart plugin, then restart the IDE
-* Open flutter-intellij project in IntelliJ (select and open the directory of the flutter-intellij repository). Build it using `Build` | `Make Project`
-* Try running the plugin; there is an existing launch config for "Flutter IntelliJ".
+  - Point it to the directory of your downloaded IntelliJ Community Edition installation (e.g, `IntelliJ IDEA CE.app/Contents` or `~/idea-IC-183.4886.37`)
+  - Change the name to `IntelliJ IDEA Community Edition`
+* One-time Dart plugin install - first-time a new IDE is installed and run you will need to install the Dart plugin. Find `Plugins` (in Settings/Preferences) and install the Dart plugin, then restart the IDE
+* Open flutter-intellij project in IntelliJ (select and open the directory of the flutter-intellij repository). Build it using `Build` | `Build Project`
```

Comment:
I'm not sure if some of these naming differences (e.g. 'Make project' vs. 'Build project') are just differences between IntelliJ versions. These were the names I found while setting up so I'm assuming they're latest, but let me know if there's some other reason for the discrepancy.

Reply:
JetBrains changes menu times sometimes. It used to be Make. I don't know about Configure but your change is better.